examples/e2e-test-dataflow/src/test/resources/e2etest/dataflow-e2e-test.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Dataflow Integration testing
+  End to End testing for the Dataflow GCS to BQ
+
+  Background:
```

Comment:
Background can be ignored as this is normally used to encapsulate the creation of common state across the scenarios